sys/xtimer/xtimer.c

```diff
@@ -67,7 +67,7 @@ void _xtimer_tsleep(uint32_t offset, uint32_t long_offset)
 
     timer.callback = _callback_unlock_mutex;
     timer.arg = (void*) &mutex;
-    timer.target = timer.long_target = 0;
+    timer.offset = timer.long_offset = 0;
```

Comment:
is this line necessary? 
`_xtimer_set64` makes sure this is not necessary, right?

Reply:
Not necessary, and thank you for the observation!